obspy/core/inventory/response.py

```diff
@@ -961,6 +961,57 @@ def is_close(a, b):
 
         return sampling_rates
 
+    @classmethod
+    def get_flat_response(cls, sensitivity=1., frequency=1.,
+                          input_units="M/S", output_units="M/S",
+                          include_pole_zero_at_zero=False):
+
+        """
+        Makes a flat response object.
+
+        :type sensitivity: float, optional
+        :param sensitivity : The sensitivity of the response, defaults to 1.
+        :type frequency: float, optional
+        :param frequency: The frequency where the sensitivity is stated,
+            defaults to 1 Hz.
+        :type input_units: string, optional
+        :param input_units: The input units going into the response object,
+            default units are "M/S"
+        :type output_units: string, optional
+        :param input_units: The output units going out of the response object,
+            default units are "M/S"
+        :type include_pole_zero_stage: bool, optional
+        :param include_pole_zero_at_zero: This option allows for a pole/zero
```

Comment:
I'm a bit confused here. Why exactly is this needed? The output from evalresp should be identical no matter if they are added or not, no?

Reply:
I included this mostly so that when I use `seismometer_simulate()` I get the exact same behavior on my synthetic as my data.  For example, the detrending, tapering etc.  This was necessary since it checks if you have poles and zeros.  It appears to still make that check on the dictionary.  I am referring to line 531: `obspy/signal/invsim.py`.

If you think this is no longer necessary, then I can definitely remove it.